scripts/bootcount_reset

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+function die() {
+	echo "$1" >&2
+	exit 1
+}
+
+/usr/bin/grub-editenv /boot/grub/grubenv set bootcount=0 && exit 0
```

Comment:
One possible concern is regarding the output that this generates. At first boot, this will essentially always output:
```
/usr/bin/grub-editenv: error: invalid environment block.
```
into the system journal, even though this will no longer be a persistent issue. A support engineer might notice this while looking over logs, precipitating an engagement with engineering.

One possible idea would be to do a `grubenv list` into a variable to check that the bootcount variable exists, and if it does not (including the case where `grubenv list` fails outright), then you do the create and set. Or maybe even more simply, redirect stdout/stderr from that first command to /dev/null since it doesn't output anything useful anyway...